test/Microsoft.AspNet.Mvc.ModelBinding.Test/Binders/ArrayModelBinderTest.cs

```diff
@@ -11,57 +11,102 @@ namespace Microsoft.AspNet.Mvc.ModelBinding.Test
     public class ArrayModelBinderTest
     {
         [Fact]
-        public async Task BindModel()
+        public async Task BindModelAsync_ValueProviderContainPrefix_Succeeds()
         {
             // Arrange
             var valueProvider = new SimpleHttpValueProvider
             {
                 { "someName[0]", "42" },
-                { "someName[1]", "84" }
+                { "someName[1]", "84" },
             };
             var bindingContext = GetBindingContext(valueProvider);
+            var modelState = bindingContext.ModelState;
             var binder = new ArrayModelBinder<int>();
 
             // Act
-            var retVal = await binder.BindModelAsync(bindingContext);
+            var result = await binder.BindModelAsync(bindingContext);
 
             // Assert
-            Assert.NotNull(retVal);
+            Assert.NotNull(result);
 
-            int[] array = retVal.Model as int[];
+            int[] array = result.Model as int[];
```

Comment:
sure; I just did a search-n-repl and didn't fix other issues in the changed lines